msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgSettings.jsp

```diff
@@ -22,23 +22,23 @@
 			<h1><h:outputText value="#{msgs.pvt_msgs_label} #{msgs.pvt_settings}" /></h1>
 			<h:messages styleClass="alertMessage" id="errorMessages" rendered="#{! empty facesContext.maximumSeverity}"/>
 			
-			    <h:panelGroup styleClass="shorttext">
-					  <h:outputLabel for="" ><h:outputText value="#{msgs.pvt_actpvtmsg1}"/></h:outputLabel>
-					</h:panelGroup>
-					<h:panelGroup >
+			    <h:panelGroup styleClass="shorttext" rendered="#{PrivateMessagesTool.instructor && PrivateMessagesTool.messagesandForums}">
```

Comment:
With the option layout="block" for the h:panelGroup a <div> will be rendered.
Without this option it will render a <span>
For this situation it might be better a <div>